tests/testthat/test-hive.R

```diff
@@ -0,0 +1,172 @@
+context("Hive translation")
+
+# Initialize simulated DB connection -------------------------------------
+
+hive_connection <- dbplyr:::simulate_hive()
```

Comment:
Yes! I hate that I saw that after submitting the PR, I would have the new tests conform before sending the PR your way